### PR TITLE
feat: improvement org member list

### LINKED ISSUE #13810: [CAL-3188] [CAL-3187] improvements to org member list
- [ ] sticky actionbar in wrong place
CleanShot 2024-02-21 at 19 27 50@2x

- [ ] make teams clickable and filterable, same as roles
CleanShot 2024-02-21 at 19 25 49@2x

- [ ] thereby, change "Roles" to "Filters" because we are adding Teams as another filter

CleanShot 2024-02-21 at 19 26 19@2x

- [ ] Change placeholder to "Filter"
- [ ] Add divider after roles and add teams
CleanShot 2024-02-21 at 19 26 55@2x

- [ ] add total members (only if it doenst slow us down like crazy)
CleanShot 2024-02-21 at 19 24 51@2x

- [ ] add pending here:
CleanShot 2024-02-21 at 19 01 52@2x

- [ ] checkboxes not aligned
image

<sub>[CAL-3188](https://linear.app/calcom/issue/CAL-3188/[cal-3187]-improvements-to-org-member-list)</sub>

### PR DESCRIPTION
**Pull request by Amit91848**
## What does this PR do?

Fixes #13810 

* [x]  sticky actionbar in wrong place
 

* [x]  make teams clickable and filterable, same as roles

https://github.com/calcom/cal.com/assets/74371312/131f30f3-0f2b-4f3a-8008-8c82e27ce36f

  

**Added another filter for teams, including it in Role would require substantial changes to the Datatable Component**.
* [ ]  thereby, change "Roles" to "Filters" because we are adding Teams as another filter
* [ ]  Change placeholder to "Filter"
* [ ]  Add divider after roles and add teams

* [x]  add total members (only if it doenst slow us down like crazy)
Screenshot 2024-02-26 at 4 06 03 PM

* [ ]  add pending here: wip
* [x]  checkboxes not aligned
Screenshot 2024-02-26 at 4 07 43 PM

[CAL-3188](https://linear.app/calcom/issue/CAL-3188/%5Bcal-3187%5D-improvements-to-org-member-list)

## Type of change

- New feature (non-breaking change which adds functionality)

## Mandatory Tasks

- [x] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

### CODE REVIEW

**Review by sean-brydon — APPROVED** [review]
Awesome thanks for this - tested locally and these work as expected

File: packages/trpc/server/routers/viewer/organizations/getTeams.handler.ts [hunk 1]
```diff
@@ -16,10 +15,6 @@ export async function getTeamsHandler({ ctx }: GetTeamsHandler) {
 
   if (!currentUser.organizationId) throw new TRPCError({ code: "UNAUTHORIZED" });
 
-  // check if user is admin of organization
-  if (!(await isOrganisationAdmin(currentUser?.id, currentUser.organizationId)))
-    throw new TRPCError({ code: "UNAUTHORIZED" });
-
```
Comment (Udit-takkar) [inline packages/trpc/server/routers/viewer/organizations/getTeams.handler.ts:22]: @Amit91848 why did you remove this?

Comment (Udit-takkar) [reply packages/trpc/server/routers/viewer/organizations/getTeams.handler.ts:22]: I'll have to confirm if we allow non admins to get the teams

Comment (Amit91848) [reply packages/trpc/server/routers/viewer/organizations/getTeams.handler.ts:22]: In order for members to filter based on teams within an organization, they would need access to view those teams. 

> I'll have to confirm if we have to allow non admins to get the teams

Will change it back to only admins being able to access and use team filters in this case.

**Review by keithwillcode — DISMISSED** [review]
Requesting changes regarding @Udit-takkar comment so this doesn’t get merged prematurely.
